test/solvers.jl

```diff
@@ -15,7 +15,7 @@
 # Detect solvers
 grb = isdir(Pkg.dir("Gurobi"))
 cpx = isdir(Pkg.dir("CPLEX"))
-mos = isdir(Pkg.dir("Mosek"))
+mos = false#isdir(Pkg.dir("Mosek"))
```

Comment:
Not intentional, I had it locally since Mosek has been failing a test in the JuMP suite for some time now.